src/deepsparse/transformers/eval_downstream.py

```diff
@@ -116,6 +117,7 @@ def mnli_eval(args):
     # load pipeline
     text_classify = pipeline(
         task="text-classification",
+        model_name=args.model_dir,
```

Comment:
This does not seem correct, `model_name` should be canonical name of the huggingface model, if model is to be initialized using a directory, I think the correct usage is to pass that directory to `model_path`, which should contain `config,json`, `tokenizer.json`, and `model.onnx`

Like the following:
```
my_model_dir
├── config.json
├── model.onnx
└── tokenizer.json
```

Reply:
Yup, I was using this wrong thanks Rahul!